workers/generic-worker/helper_test.go

```diff
@@ -374,7 +374,7 @@ func GWTest(t *testing.T) *Test {
 			// These ports are not exposed outside of the host. However, in CI they must differ from those of the
 			// generic-worker instance running the test suite.
 			LiveLogPortBase:    30583,
-			MaxTaskRunTime:     300,
+			MaxTaskRunTime:     3000,
```

Comment:
The 300 is there to avoid that we have test tasks that take too long. Can we find out why this was passing before?